src/main/scala/cromwell/backend/model/TaskDescriptor.scala

```diff
@@ -15,6 +16,7 @@ import wdl4s.values.WdlValue
   * @param runtimeAttributes Configuration needed at runtime in order to execute a task.
   */
 case class TaskDescriptor(name: String,
+                          index: Option[Int],
```

Comment:
There's an `ExecutionIndex` type for this in Cromwell, that probably should be brought over here.